djes/models.py

```diff
@@ -192,7 +202,13 @@ def mapping(self):
         return self.__class__.search_objects.mapping
 
     @classmethod
+    def is_orphaned(cls):
+        return getattr(cls.search_objects.mapping.Meta, 'orphaned', False)
+
+    @classmethod
     def get_base_class(cls):
+        if cls.is_orphaned():
```

Comment:
So an orphaned class just means that it uses a separate index apart from base Indexable index? If so, remind me what the use case is?